src/components/EventCalender/Monthly/EventCalendar.spec.tsx

```
@@ -52,16 +52,14 @@ vi.mock('types/Event/utils', async () => {
-    return new Promise((resolve) => {
-      setTimeout(resolve, ms);
-    });
-  });
-}
-
 describe('Calendar', () => {
   const onMonthChange = vi.fn();
+
+  beforeEach(() => {
+    vi.useFakeTimers({ shouldAdvanceTime: true });
+    vi.setSystemTime(new Date('2026-01-15T12:00:00Z'));
```

ESLint

[error] Avoid hardcoded date strings in tests. Use dynamic dates with dayjs instead (e.g., dayjs().add(30, "days").format("YYYY-MM-DD")).

(no-restricted-syntax)